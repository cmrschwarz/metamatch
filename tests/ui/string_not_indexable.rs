use metamatch::unquote;

fn main() {
    unquote! {
        "asdf"[2]
    }
}
