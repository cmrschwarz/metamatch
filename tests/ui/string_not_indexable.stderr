error: cannot index into `str`
 --> tests/ui/string_not_indexable.rs:5:15
  |
5 |         "asdf"[2]
  |               ^^^
